feat(docs): IOTA Identity Linked VP docs (#8659)
iotaledger/iota: docs/content/developer/iota-identity/references/iota-resource-locator.mdx

---
title: Iota Resource Locator
sidebar_label: Iota Resource Locator
description: The specification for a URL-like address that can be used to reference data on an IOTA network.
image: /img/identity/icon.png
teams:
  - iotaledger/identity
tags:
  - reference
  - identity
---

# IOTA Resource Locator (IOTA implementation of a Chain Agnostic Resource Locator)

## Abstract

An Iota Resource Locator (IRL) is a URL-like address that is used to reference resources on an
IOTA Network. 

## Specification

An IRL is a URL-like address comprised of an [IOTA Chain ID](https://github.com/ChainAgnostic/namespaces/blob/main/iota/caip2.md)

> and a relative URL (a regular URL but with only path, query, and fragment components set), where the 

Reviewer: ```suggestion
and a relative URL (a regular URL but limited to path, query, and fragment components), where the 
```
